src/main/java/org/jenkinsci/plugins/workflow/steps/StepContext.java

```diff
@@ -76,7 +76,7 @@ public abstract class StepContext implements FutureCallback<Object>, Serializabl
      * @see BodyInvoker#withContext
      * @see DynamicContext
      */
-    public abstract @Nullable <T> T get(Class<T> key) throws IOException, InterruptedException;
+    public abstract <T> T get(Class<T> key) throws IOException, InterruptedException;
 
     @Override public abstract void onSuccess(@Nullable Object result);
```

Comment:
What about this one?

Reply:
This one has not caused me any issues so far in the handful of tests I ran with the new version of SpotBugs.

Reply:
OK, let us leave it for now, but it seems that SpotBugs devs have picked an interpretation of `@Nullable` which basically makes it a confusing synonym for `@CheckForNull` which should probably just be avoided generally.